src/views/DocumentationTopic.vue

```diff
@@ -207,17 +207,25 @@ export default {
       },
       parentTopicIdentifiers,
     }) => {
-      if (!parentTopicIdentifiers.length) return references[identifier];
+      // create an end-case fallback technology
+      const fallback = { title, url: $route.path };
+      // get the reference to the current page
+      const currentPageReference = references[identifier];
+      // if there are no parent topics at all, use the current reference or a fallback one
+      if (!parentTopicIdentifiers.length) return currentPageReference || fallback;
+      // try to use the first parent topic reference, if not a technology kind
       const first = references[parentTopicIdentifiers[0]];
-      if (first.kind !== 'technologies') return first;
+      if (first && first.kind !== 'technologies') return first;
```

Comment:
I don't think this is necessary since there is already a guard for `parentTopicIdentifiers.length` above this—there is always guaranteed to be a first item here.

Reply:
Its not the same. That checks if there are any identifiers. Here I am making sure that there was a reference for that identifier at `0`.